tools/webpack.config.ts

```diff
@@ -114,12 +114,12 @@ export default (env?: string) : webpack.Configuration => {
                 },
                 // load fonts and files
                 {
-                    test: /\.(woff(2)?|ttf|eot|svg|otf)(\?v=\d+\.\d+\.\d+)?$/,
+                    test: /\.(woff(2)?|ttf|eot|svg|otf|png)(\?v=\d+\.\d+\.\d+)?$/,
                     use: [{
                         loader: 'file-loader',
                         options: {
                             name: '[name].[ext]',
-                            outputPath: 'fonts/'
+                            outputPath: 'files/'
```

Comment:
How is it possible that changing `outputPath` without changing anything else in the codebase doesn't break anything?

Reply:
Thats because this is all internal to the webpack loaders. It creates a directory and consumes the directory itself.